### DISCUSSION TITLE
What is the usage of LPTIM3?

### DISCUSSION THREAD

**Discussion by nilesh-dryad** [jefftenney/LPTIM-Tick-U5#5] [Q&A] [answered]
I was looking at the implementation more deeply and I can see that you used LPTIM1 for tickless mode. Then I found that LPTIM3 is also enabled in the IOC. But I don't find any use of it.

**Comment by jefftenney** [comment, marked as answer]
We [start LPTIM3](https://github.com/jefftenney/LPTIM-Tick-U5/blob/71a26d4f4ed8b58d23efb45a81bf80a2eb375274/Core/Src/main.c#L143) in test 3.  It provides a periodic interrupt that contributes to the "stress" in the stress test.  LPTIM3 is used only in test 3, and not in test 1 or test 2.
